GiniVision/Classes/CameraViewController.swift

```diff
@@ -245,18 +257,27 @@ public typealias CameraScreenFailureBlock = (_ error: GiniVisionError) -> ()
         camera?.stop()
     }
     
+    public override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        
+        if shouldShowFileImportToolTip {
+            showFileImportTip()
```

Comment:
Why in `viewDidAppear `? It tends to get called a lot more than `viewDidLoad`. If it's because you want to view to be present on screen before showing the tooltip, just add a delay. It would even be nice to have an explicit timeout so we can play with it to see what looks best.

Reply:
I came up with a solution for this, since we need to blur not the entire preview view but only the non-dark frame.

Reply:
Any comments?